src/status_im/ui/screens/browser/events.cljs

```diff
@@ -46,6 +47,8 @@
  :open-browser
  [re-frame/trim-v]
  (fn [{:keys [now] :as cofx} [browser]]
+   (if (some? (:history browser))
+     (reset! browser-views/forget-page-load true))
```

Comment:
Probably best to encapsulate this into a function, rather than resetting directly an atom from a different namespace.

Reply:
Done!